src/components/common/SessionApprove.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <TmPage class="approve-tran" hide-header>
```

Comment:
this doesn't need to be a page, this only adds the footer, which we don't need I think